tests/acceptance/modules/TargetLists/TargetListsCest.php

```diff
@@ -30,15 +30,15 @@ public function _before(AcceptanceTester $I)
     /**
      * @param \AcceptanceTester $I
      * @param \Step\Acceptance\ListView $listView
-     * @param \Step\Acceptance\Targets $targets
+     * @param \Step\Acceptance\TargetList $targetList
      * @param \Helper\WebDriverHelper $webDriverHelper
      *
      * As an administrator I want to view the targets module.
      */
     public function testScenarioViewTargetsModule(
         \AcceptanceTester $I,
         \Step\Acceptance\ListView $listView,
-        \Step\Acceptance\Targets $targets,
```

Comment:
Does this mean we don't have Acceptance Test for Targets or was this a mistake originally and giving us a false positive?

Reply:
Yes, the ViewTargetModule test was previously looking at the "Targets" module instead of "Targets - List".